pkg/api/plugin.go

```diff
@@ -33,6 +35,10 @@ func (m MetadataOutput) Validate() error {
 		issues = append(issues, "version field cannot be empty")
 	}
 
+	// if m.JSONSchema == "" {
```

Comment:
Can we remove this if it is not necessary?

Reply:
The JSON Schema shouldn't be required, IMO. If you have some test/dev plugins, you don't always want to have it. Additionally, not always you have input parameters.